build.gradle

```diff
@@ -145,6 +145,9 @@ dependencies {
     androidTestCompile 'com.android.support:support-annotations:25.4.0'
     provided 'com.squareup.dagger:dagger-compiler:1.2.5'
     compile 'com.github.wseemann:FFmpegMediaMetadataRetriever:1.0.14'
+    debugCompile 'com.readystatesoftware.chuck:library:1.1.0'
+    releaseCompile 'com.readystatesoftware.chuck:library-no-op:1.1.0'
+    betaReleaseCompile 'com.readystatesoftware.chuck:library-no-op:1.1.0'
```

Comment:
In here for debug app, the dependency is `library` whereas for release and betaRelease variants the dependency is `library-no-op`. The no-op variant of the library contains no actual code just the empty function definitions. So that the code does not crash and also that unused code is not compiled into the apk. So whenever we build the debug app everyone will see it, but in release app it will have no effect.